package.json

```diff
@@ -109,16 +104,16 @@
     "jest": "^24.1.0",
     "lerna": "^3.10.7",
     "lint-staged": "^9.0.0",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
     "markdown-spellcheck": "^1.3.1",
     "metro-react-native-babel-preset": "^0.55.0",
     "prompt": "^1.0.0",
     "prop-types": "^15.5.10",
     "react": "16.8.3",
```

Comment:
You also need to update `react` and `react-dom` dependencies.

Reply:
good catch, done!